pkgs/applications/networking/sync/desync/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "desync-${version}";
+  version = "0.3.0";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/folbricht/desync";
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://github.com/folbricht/desync";
+    sha256 = "1h2i6ai7q1mg2ysd3cnas96rb8g0bpp1v3hh7ip9nrfxhlplyyda";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    description = "Content-addressed binary distribution system";
+    longDescription = "An alternate implementation of the casync protocol and storage mechanism with a focus on production-readiness";
+    homepage = https://github.com/folbricht/desync;
+    license = licenses.bsd3;
+    maintainers = [ maintainers.chaduffy ];
```

Comment:
Please add `platforms = platforms.unix;` to specify supported platforms.
`unix` includes linux and darwin.

Reply:
Platform is not limited to unixlikes -- Windows is supported (well, may have been broken in 0.3.0, but is an *intended* target; has been supported in past releases, is working again on master).

Reply:
We shouldn't include platforms that don't work in this version. This can be changed in a later version of course.

Reply:
By convention, `meta.platforms` should be specified  (although omitting it doesn't cause an error). 
It can be set to `platforms.all` to mean no restriction.